File: shoebill_plugins/GaragePlugin/src/main/java/lt/ltrp/garage/event/GarageEvent.java
package lt.ltrp.garage.event;

import lt.ltrp.event.property.PropertyEvent;
import lt.ltrp.object.Garage;

/**
 * @author Bebras
 *         2016.04.20.
 */
public class GarageEvent extends PropertyEvent {


    public GarageEvent(Garage garage) {
        super(garage);
    }

    @Override
    public Garage getProperty() {
        return (Garage)super.getProperty();
    }
}

File: shoebill_plugins/GaragePlugin/src/main/java/lt/ltrp/event/GarageEvent.kt
package lt.ltrp.event

import lt.ltrp.`object`.Garage
import lt.ltrp.event.property.PropertyEvent

/**
 * Created by Bebras on 2016-10-06.
 */
abstract class GarageEvent(var garage: Garage): PropertyEvent(garage) {
}